config/default.ini

```diff
@@ -4,7 +4,7 @@ name:          Scratch2Catrobat Converter
 short_name:    S2CC
 version:       0.10.0
 build_name:    Aegean cat
-build_number:  1028
+build_number:  1030-STCC-251_jython_libs
```

Comment:
Any idea why the build number is incremented again and the branch name added? Both should no longer happen and right now I don't see what changed it. Please have a look at that.

Reply:
I tested something on the master branch and forgot to delete the hook afterwards.